README.md

```diff
@@ -116,7 +116,8 @@ place for people (and asdf itself) to look for plugins.
 | SBT       | [gabrielelana/asdf-sbt](https://github.com/gabrielelana/asdf-sbt) | [![Build Status](https://travis-ci.org/gabrielelana/asdf-sbt.svg?branch=master)](https://travis-ci.org/gabrielelana/asdf-sbt)
 | Scala     | [mtatheonly/asdf-scala](https://github.com/mtatheonly/asdf-scala)| [![Build Status](https://travis-ci.org/mtatheonly/asdf-scala.svg?branch=master)](https://travis-ci.org/mtatheonly/asdf-scala)
 | Serf      | [Banno/asdf-hashicorp](https://github.com/Banno/asdf-hashicorp) | [![Build Status](https://travis-ci.org/Banno/asdf-hashicorp.svg?branch=master)](https://travis-ci.org/Banno/asdf-hashicorp)
-| Shfmt     | [Luizm/asdf-shfmt](https://github.com/luizm/asdf-shfmt)         | [![Build Status](https://travis-ci.org/Luizm/asdf-shfmt.svg?branch=master)](https://travis-ci.org/luizm/asdf-shfmt)
+| Shfmt     | [luizm/asdf-shfmt](https://github.com/luizm/asdf-shfmt)         | [![Build Status](https://travis-ci.org/luizm/asdf-shfmt.svg?branch=master)](https://travis-ci.org/luizm/asdf-shfmt)
+| Shellcheck | [luizm/asdf-shellcheck](https://github.com/luizm/asdf-shellcheck) | [![Build Status](https://travis-ci.org/luizm/asdf-shellcheck.svg?branch=master)](https://travis-ci.org/luizm/asdf-shellcheck)
```

Comment:
These plugins are alphabetically sorted. `e` comes before `f`, so I think `shellcheck` should come before `shfmt`